src/weathergen/utils/compare_run_configs.py

```diff
@@ -1,3 +1,12 @@
+#!/usr/bin/env -S uv run
+# /// script
+# dependencies = [
+#   "tabulate",
+# ]
+# [tool.uv.sources]
+# TBD
```

Comment:
@tjhunter I can test this later. Maybe you can suggest what line 7 needs to be here.